interfaces/builtin/pulseaudio.go

```diff
@@ -23,27 +23,157 @@ import (
 	"github.com/snapcore/snapd/interfaces"
 )
 
-const pulseaudioConnectedPlugAppArmor = `
-/etc/pulse/ r,
-/etc/pulse/* r,
+var pulseaudioConnectedPlugAppArmor = []byte(`
+capability dac_override,
```

Comment:
Pulseaudio tries to switch to a different user which is configured at compile time (root, root in our case). I wasn't sure if we want this left in or better disable the setuid/setgid call in the code.